perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/MembersManagerMethod.java

```diff
@@ -290,6 +292,23 @@ public String call(ApiCaller ac, Deserializer params) throws PerunException {
 		}
 	},
 
+	/*#
+	 * Gets map of sponsored members and lists of their sponsors for each of them.
+	 *
+	 * @param vo int id of virtual organization from which are the sponsored members chosen
+	 * @throw VoNotExistsException if given VO does not exist
+	 * @throw PrivilegeException if not VOADMIN, VOOBSERVER, PERUNOBSERVER or SPONSOR
```

Comment:
We do not usually specify ``PrivilegeException`` in the RPC docs, since its added automatically by the docs parsing tool.

Reply:
I deleted this `PrivilegeException` from the RPC.